src/EventListener/AssetsInstallCommandListener.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\EventListener;
+
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Event\ConsoleTerminateEvent;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * This listener extends `assets:install` command when SonataCoreBundle will be not register. Files from `Resources/private/SonataCoreBundleAssets`
+ * will be copy with the same result like SonataCoreBundle is register.
+ *
+ * This class should be remove when support for Bootstrap 3 will be ended or assets system will be remove in favor for encore webpack.
```

Comment:
This class should be removed on Sonata 4, all assets belong here. On Sonata 4 we will decide if we remove bootstrap 3 or what, but it is not dependent, right?

Reply:
https://github.com/sonata-project/SonataAdminBundle/pull/6096#discussion_r437312790

This `sonatacore` assets are needed in whole sonata 3 (bootstrap 3). Sonata have layouts in 2 places `AminBundle` and `PageBundle` (this require AdminBundle). This mean move this assets to AdminBundle is mandatory.

Existing projects using this assets. We should allow use it in Sonata 4 where should be two assets system - deprecated assets for keep BC and new webpack. 

This class(and sonatacore assets) should be remove in Sonata 5 then or we must add webpack in Sonata 3.